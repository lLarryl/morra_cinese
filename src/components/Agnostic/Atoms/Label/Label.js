import React from 'react';

export const Label = ({ value }) => (
  <div>
    <span>{value}</span>
  </div>
);
