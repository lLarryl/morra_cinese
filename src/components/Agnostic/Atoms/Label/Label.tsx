import * as React from 'react';

export const Label = ({ value }: {value: string}) => (
  <div>
    <span>{value}</span>
  </div>
);
