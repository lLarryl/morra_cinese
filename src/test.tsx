import { Label } from './components/Agnostic/Atoms/Label/Label';
import React from 'react';
import renderer from 'react-test-renderer';

test('Label has text', () => {
    const component = renderer.create(
      <Label value="testo di test" />,
    );
    let result = component.toJSON();
    expect(result).toMatchSnapshot();
});
