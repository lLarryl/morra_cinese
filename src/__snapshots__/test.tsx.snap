// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label has text 1`] = `
<div>
  <span>
    testo di test
  </span>
</div>
`;
